jsfuck.js

```diff
@@ -63,15 +63,15 @@
     'I':   '(Infinity+"")[0]',
     'J':   USE_CHAR_CODE,
     'K':   USE_CHAR_CODE,
-    'L':   'Function("return Object.entries(Error)")()[0][0][10]', // []["fill"]["constructor"]("return Object.entries(Error)")()[0][0][10] returns a capital L
+    'L':   '(""+Object["entries"](Function("return Error")()))[10]',
     'M':   '(true+Function("return Date")()())[30]',
     'N':   '(NaN+"")[0]',
     'O':   '(+[]+Object)[10]',
     'P':   USE_CHAR_CODE,
     'Q':   USE_CHAR_CODE,
     'R':   '(+[]+RegExp)[10]',
     'S':   '(+[]+String)[10]',
-    'T':   '(NaN+Function("return Date")()())[30]',
+    'T':   '(""+Object["entries"](Function("return Error")()))[5]',
```

Comment:
You should use `(false+Object["entries"](Function("return Error")()))[10]` here. It actually saves 15 characters by avoiding using the number 5. "10" is much cheaper, it's just "1" + "0".

Reply:
Ou, of course, that seems reasonable. Here you go.